### PR TITLE
Bring over two changes from mscodehub: adding support for class libs and changing NuGet folder structure per request.

### PR DESCRIPTION
**Pull request by rohanp-msft**

### CODE REVIEW

**Review by axelandrejs — APPROVED** [review]

File: dev/MRTCore/TestApps/winui_class_lib/winui_class_lib.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.ApplicationModel.Resources;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace winui_class_lib
```
Comment (ptorr-msft) [inline dev/MRTCore/TestApps/winui_class_lib/winui_class_lib.cs:14]: This is not the naming convention for .NET or WinRT types. It is a C++ naming convention.

I would suggest the namespace be (eg) `SampleLibrary` and the type name be `SampleType` (if indeed the desire is to have a generic sample with no meaningful behaviour other than the "get a string" method.

File: dev/MRTCore/azure-pipelines.yml [hunk 2]
```diff
@@ -168,7 +168,7 @@ jobs:
     inputs:
       SourceFolder: '$(MRTBinariesDirectory)\Release\AnyCPU\Microsoft.ApplicationModel.Resources.Projection'
       Contents: |
-        Microsoft.ApplicationModel.Resources.winmd
+        net5.0\Microsoft.ApplicationModel.Resources.winmd
```
Comment (huichen123) [inline dev/MRTCore/azure-pipelines.yml:171]: >net5.0 [](start = 8, length = 6)

need to see if this builds

Comment (huichen123) [reply dev/MRTCore/azure-pipelines.yml:171]: nope

---
In reply to: [491236518](https://github.com/microsoft/ProjectReunion/pull/199#discussion_r491236518) [](ancestors = 491236518)

Comment (rohanp-msft) [reply dev/MRTCore/azure-pipelines.yml:171]: On my machine, I see this:

S:\feature_work\reunion_mrt_tooling\MRTCore\mrt\AnyCPU\Debug\Microsoft.ApplicationModel.Resources.Projection\net5.0\Microsoft.ApplicationModel.Resources.winmd

Perhaps the source dir needs to be changed here to
`SourceFolder: '$(MRTBinariesDirectory)\AnyCPU\ReleaseMicrosoft.ApplicationModel.Resources.Projection'`

Also, I wonder why this has worked fine in the CI builds in MSCodeHub...

File: dev/MRTCore/TestApps/winui_class_lib/winui_class_lib.cs (hunk 1 above)
Comment (rohanp-msft) [reply dev/MRTCore/TestApps/winui_class_lib/winui_class_lib.cs:14]: Alright, will fix this in a future, separate change.

Comment (rohanp-msft) [reply dev/MRTCore/TestApps/winui_class_lib/winui_class_lib.cs:14]: Will address this in a future, separate change.

File: dev/MRTCore/azure-pipelines.yml (hunk 2 above)
Comment (rohanp-msft) [reply dev/MRTCore/azure-pipelines.yml]: Fixed.

**Pull request merged**